src/drive/web/modules/views/Drive/index.jsx

```diff
@@ -89,14 +89,8 @@ const DriveView = ({ folderId, router, children }) => {
           currentFolderId={currentFolderId}
           navigateToFolder={navigateToFolder}
         />
-        {false && (
-          <Toolbar
-            folderId={null}
-            canUpload={true}
-            canCreateFolder={true}
-            disabled={false}
-          />
-        )}
+        {/* TODO do not have props hardcoded */}
+        <Toolbar canUpload={true} canCreateFolder={true} disabled={false} />
```

Comment:
we don't need to pass `folderId` ? You got it from the `getCurrentFolderId` selector ?

Reply:
Yes.